modules/common/src/main/scala/notebook/front/widgets/package.scala

```diff
@@ -17,6 +17,11 @@ import widgets.Defaults.DEFAULT_MAX_POINTS
  * This package contains primitive widgets that can be used in the child environment.
  */
 package object widgets {
+
+  type RenderableItems = Seq[MagicRenderPoint]
```

Comment:
or should we use 
```scala
type RenderableItem = MagicRenderPoint

type RenderableItems = Seq[RenderableItem] // could use Seq[RenderableItem] in fn definitions
``` ?

for very far refactoring, I don't like very much to have `RenderableItems = Seq[MagicRenderPoint]`, because it don't give access to the schema.